geoapps/inversion/driver.py

```diff
@@ -368,17 +368,23 @@ def start_inversion(filepath=None, **kwargs):
 
     if inversion_type == "magnetic vector":
         from .potential_fields import MagneticVectorParams
+        from .potential_fields.magnetic_vector.constants import validations
 
+        input_file = InputFile.read_ui_json(filepath, validations=validations)
```

Comment:
Move this down after the logicals and wait to instantiate the Params.

Reply:
actually can't move them down since we're capturing a different validations at the if block scope into a specific params.

Reply:
still all name validations though?